x-pack/plugins/observability_onboarding/public/assets/standalone_agent_setup.sh

```diff
@@ -2,27 +2,120 @@
 
 API_KEY_ENCODED=$1
 API_ENDPOINT=$2
+AUTO_DOWNLOAD_CONFIG=$3
 
 updateStepProgress() {
-  echo "  GET $API_ENDPOINT/step/$1?status=$2"
+  local STEPNAME="$1"
+  local STATUS="$2" # "incomplete" | "complete" | "disabled" | "loading" | "warning" | "danger" | "current"
   curl --request GET \
-    --url "$API_ENDPOINT/step/$1?status=$2 2023-05-24" \
-    --header "Authorization: ApiKey $API_KEY_ENCODED" \
+    --url "${API_ENDPOINT}/custom_logs/step/${STEPNAME}?status=${STATUS}" \
+    --header "Authorization: ApiKey ${API_KEY_ENCODED}" \
     --header "Content-Type: application/json" \
-    --header "kbn-xsrf: true"
-  echo ""
+    --header "kbn-xsrf: true" \
+    --output /dev/null \
+    --no-progress-meter
```

Comment:
What is this flag used for?

Reply:
This prevents the download progress summary from being printed for every progress update. Without it, there is a lot of unnecessary noise in the standard output as progress updates are pushed.